EduVPN-macOS/Controllers/AppDelegate.swift

```diff
@@ -22,7 +22,9 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         // Insert code here to initialize your application
-        UserDefaults.standard.register(defaults: NSDictionary(contentsOf: Bundle.main.url(forResource: "Defaults", withExtension: "plist")!)! as! [String: Any]) //swiftlint:disable:this force_cast
+        if let defaultsPlist = Bundle.main.url(forResource: "Defaults", withExtension: "plist"), let defaultsDict = NSDictionary(contentsOf: defaultsPlist) as? [String: Any] {
+            UserDefaults.standard.register(defaults: defaultsDict)
+        }
```

Comment:
Sometimes forced unwraps are good. If somehow the plist is missing we should notice that at first launch of the app, not fail silently like it does now.

Reply:
Technically true. But I would prefer the fatal state to be programmed explicitly. With a helpful fatal error message.